code/API_definitions/NetworkSliceBooking.yaml

```diff
@@ -0,0 +1,293 @@
+    delete:
+      tags:
+        - NSB Sessions
+      summary: Delete a NSB session
+      description: Deleting NSB session
+      operationId: deleteSession
+      parameters:
+        - name: sessionId
+          in: path
+          description: Session ID that was obtained from the createSession operation
+          required: true
+          schema:
+            $ref: "#/components/schemas/SessionId"
+      responses:
+        "204":
+          description: Session deleted
+    get:
+      tags:
+        - NSB Sessions
+      summary: Get NSB session information
+      description: Querying for NSB session resource information details
+      operationId: getSession
+      parameters:
+        - name: sessionId
+          in: path
+          description: Session ID that was obtained from the createSession operation
+          required: true
+          schema:
+            $ref: "#/components/schemas/SessionId"
+      responses:
+        "200":
+          description: Contains information about active session
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SessionInfo"
+components:
+  schemas:
+    SessionId:
+      description: Session ID in UUID format
+      type: string
+      format: uuid
+
+    SessionInfo:
+      description: Session related information.
+      allOf:
+        - $ref: "#/components/schemas/CreateSession"
+        - type: object
+          properties:
+            sessionId:
+              $ref: "#/components/schemas/SessionId"
+            result:
+              type: string
+              enum:
+                - Success
+                - Fail
+          required:
+            - result
+
+
+
+    CreateSession:
+      description: Attributes required to create a session
+      type: object
+      properties:
+        ServiceTime:
+          $ref: "#/components/schemas/ServiceTime"
+        ServiceArea:
+          $ref: "#/components/schemas/Area"
+        SLAProfile:
+          $ref: "#/components/schemas/SLAProfile"
+      required:
+        - ServiceTime
+        - ServiceArea
+        - SLAProfile
+
+    SLAProfile:
+      type: object
+      properties:
+        MaxNumofTerminals:
+          $ref: "#/components/schemas/NumofTerminals"
+        DLThroughputPerTerminal:
+          $ref: "#/components/schemas/Throughput"
+        ULThroughputPerTerminal:
+          $ref: "#/components/schemas/Throughput"
+        DLLatency:
+          $ref: "#/components/schemas/Latency"
+        ULLatency:
+          $ref: "#/components/schemas/Latency"
+
+
+    Latency:
+      type: integer
+      example: 10
+      description: |
+        An attribute specifies the required DL packet transmission latency (millisecond) 
+        through the RAN, CN, and TN part of 5G network.
+      format: int32
+
+
+    NumofTerminals:
+      type: integer
+      example: 5
+      description: Number of terminals
+      format: int32
+      minimum: 1
+      maximum: 20

```

Comment:
One more thing. Just let me ask you.
Device access management is mentioned in api design [#20](https://github.com/camaraproject/NetworkSliceBooking/pull/20).
In this PR, the API can specify the number of terminals but cannot specify which terminals will be allowed to access the slice.
Do you have a plan to add more parameter related to device access management or create new device access management API later?

Reply:
Yes Masa, based on Thorsten's suggestion, we'd better need to make device access management function independently.  Basically there is one slice resource reservation function, and another device access management function.